app/grandchallenge/direct_messages/emails.py

```diff
@@ -73,11 +77,10 @@ def send_new_unread_direct_messages_email(
         new_senders=oxford_comma(new_senders),
         url=reverse("direct-messages:conversation-list"),
     )
-    site = Site.objects.get_current()
     send_standard_email_batch(
         site=site,
         subject=subject,
         markdown_message=msg,
         recipients=[user],
-        subscription_type=EmailSubscriptionTypes.SYSTEM,
+        subscription_type=EmailSubscriptionTypes.NOTIFICATION,
```

Comment:
I noticed that we check the notification email preference when determining whether or not to email users about unread messages. I wasn't aware of that before, and wrongly classified them as system messages. It's also not clear from the help text of the model field that this is the case. I wonder if a seperate email preference setting would make sense for direct messages? That could be done in a seperate PR. Else I will just adapt the help text.

Reply:
Good catch! I think help text modification is fine.